OxfordBodleian/Juel-Jensen/BDLaethg34.xml

```diff
@@ -112,10 +112,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             <list>
                                 <item xml:id="e1">
                                     <desc>A white sticker is glued to the inner side of the front board. The name of <persName ref="PRS5782JuelJen"/> is printed on it
-                                        (<foreign xml:lang="gez">ቤንት፡ ዩውል፡ የንሰን።</foreign>) and the signatures "M Aeth. 13" and "MS. Aeth. g. 34" are written on it in pencil.
-                                        An additional sticker, with the signature "Ms 13" on it, is glued below it. Below the stickers, hardly legible notes in pencil, presumably giving
-                                    a brief summary of the manuscript's contents.</desc>
-                                   <!-- quotation marks-->
+                                        (‘<foreign xml:lang="gez">ቤንት፡ ዩውል፡ የንሰን።</foreign>’) and the signatures ‘M Aeth. 13’ and ‘MS. Aeth. g. 34’ are written on it in pencil.
+                                        An additional sticker, with the signature ‘Ms 13’ on it, is glued below it. Below the stickers, hardly legible notes in pencil, presumably giving
+                                    a brief summary of the manuscript’s contents.</desc>
```

Comment:
One could have a reference to other manuscripts in the collection which have the same kind of summary. Or is it unnecessary?